index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<link rel="stylesheet" href="styles/main.css">
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>AC GILLETTE</title>
+  </head>
+  <header>
+    <h1>AC GILLETTE</h1>
+    <nav>
+      <a href="index.html">Home</a>
+      <a href="about.html">About</a>
+      <a href="projects.html">Projects</a>
+      <a href="writing.html">Writing</a>
+    </nav>
+  </header>
+  <body>
+    <section id="synopsis">
+      <img src="assets/skull2.gif" alt="spinning skull" />
+
+      <p>
+        Hi I'm AC! I'm currently a software development student at Ada Developer's Academy.
+      </p>
+
+      <p>
+        When I'm not doing that, I like making weird internet art and watching a lot of scifi.
+      </p>
+      <img src="assets/skull2.gif" alt="spinning skull" />
+    </section>
+
+    <section class="options">
+      <section class="link" id ="projects">
+        <a href="projects.html"><h2>HOT PROJECTS</h2></a>
+
+      </section>
+      <section class="link" id="me">
+        <a href="about.html"><h2>WHO IS THIS PERSON</h2></a>
+
+      </section>
+      <section class="link" id="writing">
+        <a href="writing.html"><h2>COOL WRITING</h2></a>
+
+      </section>
+    </section>
+
+  </body>
+  <footer>
+    <p>
```

Comment:
Great it looks like you've made the choice to have you're footer be at the same place on each page.